contracts/src/v0.8/vrf/VRFV2PlusWrapper.sol

```diff
@@ -369,15 +369,15 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     uint256 price = calculateRequestPriceNativeInternal(_callbackGasLimit, tx.gasprice);
     require(msg.value >= price, "fee too low");
     require(_numWords <= s_maxNumWords, "numWords too high");
-
-    requestId = COORDINATOR.requestRandomWords(
-      s_keyHash,
-      SUBSCRIPTION_ID,
-      _requestConfirmations,
-      _callbackGasLimit + eip150Overhead + s_wrapperGasOverhead,
-      _numWords,
-      true
-    );
+    VRFV2PlusClient.RandomWordsRequest memory req = VRFV2PlusClient.RandomWordsRequest({
+      keyHash: s_keyHash,
+      subId: SUBSCRIPTION_ID,
+      requestConfirmations: _requestConfirmations,
+      callbackGasLimit: _callbackGasLimit + eip150Overhead + s_wrapperGasOverhead,
+      numWords: _numWords,
+      extraArgs: VRFV2PlusClient._argsToBytes(VRFV2PlusClient.ExtraArgsV1({nativePayment: true}))
```

Comment:
This is interesting, `VRFV2PlusClient` doesn't need to be imported?

Reply:
yeah it's a transitive dependency via `VRFConsumerBaseV2Plus`